packages/common/src/api/tan-query/wallets/useSendTokens.ts

```diff
@@ -111,6 +115,28 @@ export const useSendTokens = ({ mint }: { mint: string }) => {
         })
       }
 
+      // For AUDIO, also optimistically update the audio balance queries
+      if (isAudioMint && currentUser?.spl_wallet) {
+        // Update both staked and non-staked balance queries for the SOL wallet
+        for (const includeStaked of [true, false]) {
```

Comment:
at this point i think we should create a helper here since we have this in a few spots? @DejayJD maybe can use for claim fees too?